src/components/FeatureViews/Search/Search.scss

```diff
@@ -84,14 +84,24 @@
   }
 }
 
-#select-primary-filter {
-  height: 22px;
-  border: 1px solid #eeeeee;
+.MuiOutlinedInput-root {
+  position: relative;
+  border-radius: 3px 0px 0px 3px !important; 
+}
+
+#mui-component-select-primary-filter {
+  height: 20px;
+  border: 1px solid #eee;
   background: #ffffff;
   border-radius: 3px 0px 0px 3px;
   padding: 11px 20px 7px 15px;
 }
 
+
+.MuiSelect-iconOutlined {
+  right: 0px !important;
+}
+
```

Comment:
All of these styles (.MuiOutlinedInput-root, #mui-component-select-primary-filter, .MuiSelect-iconOutlined) should be using JSS instead of CSS.  Refer to SchedulerUtils.js to see how to create a custom styled input/select with JSS. 

You should NEVER be writing ".Mui..." in the styling sheets because that will be applied to ALL Material UI components. If you need to select a specific item, always make an id or a class. (Also, if you're using `!important` it probably means there's a better way of doing this.)